### DISCUSSION TITLE
Cant't find unity student pack

### DISCUSSION THREAD

**Discussion by Moosaminhaj** [community/community#49050] [GitHub Education] [answered]
i enrolled for this to have a unity student plan. But its missing from the packages.

**Comment by samus-aran** [comment, marked as answer]
Hi @Moosaminhaj 👋 

We're happy you're here. Unity is currently not part of GitHub Student Developer Pack. Here is a full list of the [Global Campus Student Developer Pack benefits](https://github.com/education/Student-Developer-Pack-Current-Partners-FAQ)
